lib/query-builders/filters/by-action.js

```diff
@@ -1,3 +1,9 @@
 module.exports = action => query => {
+  if (action === 'change-licence-holder') {
+    return query.andWhere(builder => {
+      builder.whereJsonSupersetOf('data', { action: 'update' });
+      builder.whereRaw(`data->'modelData'->>'licenceHolderId' <> data->'data'->>'licenceHolderId'`);
```

Comment:
```suggestion
      builder.whereRaw(`data->'modelData'->>'licenceHolderId' != data->'data'->>'licenceHolderId'`);
```

It's equivalent, but one I immediately understand and the other breaks my brain.